src/schemes/boundary/dummy_particles/dummy_particles.jl

```diff
@@ -299,8 +334,10 @@ end
     boundary_model.pressure[particle] = max(boundary_model.state_equation(density), 0.0)
 end
 
-function compute_pressure!(boundary_model, ::AdamiPressureExtrapolation, system, v, u,
-                           v_ode, u_ode, semi)
+function compute_pressure!(boundary_model,
+                           boundary_density_calc::Union{AdamiPressureExtrapolation,
+                                                        BernoulliPressureExtrapolation},
```

Comment:
```suggestion
                           ::Union{AdamiPressureExtrapolation, BernoulliPressureExtrapolation},
```

Reply:
Doesn't fit

Reply:
the variable `boundary_density_calc` is unused